internal/options/options.go

```diff
@@ -37,6 +38,10 @@ func ParseOptions() (Options, error) {
                         //host, port, dbname, user, password),
                        "",
                    "Connection string in Postgres format")
+    flag.StringVar(&cfg.Key,
+                    "k","",
+                    "Sing key")
+                    
```

Comment:
```suggestion
    flag.StringVar(&cfg.Key,  "k", "", "Sing key")     
```
тяжело читается если честно. Обычно либо все на одной строке, либо каждый параметр на новой